pcsx2/SPU2/Windows/Config.cpp

```diff
@@ -192,8 +184,7 @@ void CheckOutputModule(HWND window)
 {
 	OutputModule = SendMessage(GetDlgItem(window, IDC_OUTPUT), CB_GETCURSEL, 0, 0);
 	const bool IsConfigurable =
-		mods[OutputModule] == PortaudioOut ||
-		mods[OutputModule] == WaveOut;
+		mods[OutputModule] == PortaudioOut;
```

Comment:
You can move this on the same row as IsConfigurable

Reply:
done :)